mllib/src/main/scala/org/apache/spark/mllib/fpm/AssociationRules.scala

```diff
@@ -61,6 +61,18 @@ class AssociationRules private[fpm] (
    */
   @Since("1.5.0")
   def run[Item: ClassTag](freqItemsets: RDD[FreqItemset[Item]]): RDD[Rule[Item]] = {
+    run(freqItemsets, Map.empty[Item, Long])
+  }
+
+  /**
+   * Computes the association rules with confidence above `minConfidence`.
+   * @param freqItemsets frequent itemset model obtained from [[FPGrowth]]
+   * @return a `Set[Rule[Item]]` containing the association rules. The rules will be able to
```

Comment:
This doesn't agree with the signature

Reply:
nice catch, thanks, I'll correct also the original one I took this from.